quipucords/api/common/serializer.py

```diff
@@ -105,13 +105,3 @@ def to_representation(self, value):
         if value == "":
             return value
         return json.loads(value)
-
-
-class ForcedListSerializer(ListSerializer):  # pylint: disable=abstract-method
```

Comment:
With this, shouldn't `ListSerializer` be removed from line 19 ?

```
from rest_framework.serializers import (
    ChoiceField,
    Field,
    ListSerializer,
    ModelSerializer,
    ValidationError,
)
```

Reply:
yep, you are right.

we definitelly need to update the way our linter is executed.

(it's only checking git diff to speed up things, probably that's why it's not complaining about it)